cni/telemetry/service/telemetrymain.go

```diff
@@ -97,6 +97,14 @@ func setTelemetryDefaults(config *telemetry.TelemetryConfig) {
 	if config.BatchSizeInBytes == 0 {
 		config.BatchSizeInBytes = defaultBatchSizeInBytes
 	}
+
+	if config.GetEnvRetryCount == 0 {
+		config.GetEnvRetryCount = 2
+	}
+
+	if config.GetEnvRetryWaitTimeInSecs == 0 {
+		config.GetEnvRetryWaitTimeInSecs = 3
```

Comment:
change const ( 2 and 3 ) to something similar to what you have above? e.g. - defaultBatchSizeInBytes